slicer.go

```diff
@@ -725,3 +739,201 @@ func (f *File) addWorkbookSlicerCache(slicerCacheID int, URI string) error {
 	wb.ExtLst = &xlsxExtLst{Ext: strings.TrimSuffix(strings.TrimPrefix(string(extLstBytes), "<extLst>"), "</extLst>")}
 	return err
 }
+
+// GetSlicers provides the method to get all slicers in a worksheet by a given
```

Comment:
Good job. I think the `GetSlicers` function is almost done, please add unit tests for this.

Reply:
Thanks for the review. The unit tests for the `GetSlicers` function has been added.